### DISCUSSION TITLE
A new variant of this repository -> Domain driven design

### DISCUSSION THREAD

**Discussion by shamscorner** [antfu/vitesse#231] [General]
I have forked your repo and modified it a little bit to support the modular architecture pattern (Domain Driven Development). You can see the [details here](https://www.shamscorner.com/blog/vue-3-vite-modular-architecture-vitesse).

Here is the [github repo link](https://github.com/shamscorner/vitesse-stackter-clean-architect)

**Comment by antfu** [comment]
Looks cool! Happy to receive a PR adding it to the variant list. 👍
